src/settings/font.rs

```diff
@@ -132,3 +132,28 @@ impl From<FontSettings> for FontOptions {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_font_settings() {
```

Comment:
Can you also add a test for specifing a single font, to make sure that case didn't break instead?